examples/using-js-search/.gitignore

```diff
@@ -0,0 +1,69 @@
+# Logs
```

Comment:
I think these .gitignores and .prettierrcs can all go away in the example, it would be more consistent with the other examples: https://github.com/gatsbyjs/gatsby/tree/master/examples